app/javascript/chat/ChatChannelSettings/LeaveMembershipSection.jsx

```diff
@@ -13,20 +14,20 @@ const LeaveMembershipSection = ({
     <div className="crayons-card p-4 grid gap-2 mb-4 leave_membership_section">
       <h3>Danger Zone</h3>
       <div>
-        <button
+        <Button
           className="crayons-btn crayons-btn--danger leave_button"
```

Comment:
Same as mentioned above and for a danger button, use the `variant` prop.

```suggestion
          className="leave_button"
          variant="danger"
```